File: baekjoon/1057_토너먼트/토너먼트.py
# 실버4: https://www.acmicpc.net/problem/1057

total, jimin, hanso = map(int, input().split())
result = 0

while jimin != hanso:
  jimin = (jimin + 1) // 2
  hanso = (hanso + 1) // 2
  result += 1

print(result)

File: baekjoon/silver/4_1057_토너먼트/토너먼트.js
//실버4: https://www.acmicpc.net/problem/1057

const fs = require("fs");
const file = process.platform === "linux" ? "dev/stdin" : "./text.txt";
const input = fs.readFileSync(file).toString().trim().split(" ").map(Number);
let [total, jimin, hanso] = input;
let result = 0;

while (jimin !== hanso) {
  jimin = Math.floor((jimin + 1) / 2);
  hanso = Math.floor((hanso + 1) / 2);
  result = result + 1;
}
console.log(result);
